ls1intum/Artemis
# `Programming exercises`: Fix Swift exercise creation

### Checklist
#### General


#### Changes affecting Programming Exercises

### Motivation and Context
Closes #6436

### Description
There are some template directories for Swift exercises that are wrongly detected to be files. Therefore copying fails for those. This PR fixes the ignore-mechanism that matches those files and ignores them in the copying process.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create a new Swift programming exercise.
2. Go to the online editor: Check that the placeholders as shown in the screenshot are replaced in the three exercise repositories by their intended package name.

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage
A new test was added that checks the intended behaviour.

### Screenshots
before:
![Screenshot 2023-04-11 at 20-34-40 Code Editor](https://user-images.githubusercontent.com/64250573/231258515-5f1a95a8-0e66-4c8a-ad69-37a9c9974d00.png)
